lib/utils.js

```diff
@@ -3,21 +3,9 @@
 var bind = require('./helpers/bind');
 var isBuffer = require('is-buffer');
 
-/*global toString:true*/
-
 // utils is a library of generic helper functions non-specific to axios
 
-var toString = Object.prototype.toString;
-
-/**
- * Determine if a value is an Array
- *
- * @param {Object} val The value to test
- * @returns {boolean} True if value is an Array, otherwise false
- */
-function isArray(val) {
-  return toString.call(val) === '[object Array]';
-}
+var _toString = Object.prototype.toString;
```

Comment:
Changed this to start with `_`, to avoid having to place an eslint directive.

It also functions as a callout to potentially hacky / legacy code